zipline/algorithm.py

```diff
@@ -410,9 +410,6 @@ def noop(*args, **kwargs):
         # target/delta of the capital changes, along with values
         self.capital_changes = kwargs.pop('capital_changes', {})
 
-        # A dictionary of the actual capital change deltas, keyed by timestamp
-        self.capital_change_deltas = {}
```

Comment:
Are these the same values that are now being yielded out, so we don't track them here anymore?